app/src/pages/outside/loginPage/pageBlocks/changePasswordBlock/changePasswordForm/changePasswordForm.jsx

```diff
@@ -132,25 +132,29 @@ export class ChangePasswordForm extends PureComponent {
                 />
               }
             >
-              <FieldErrorHint>
+              <FieldErrorHint provideHint={false}>
                 <InputOutside
                   type={'password'}
                   icon={PasswordIcon}
                   maxLength={'256'}
                   placeholder={formatMessage(placeholders.newPassword)}
+                  hasDynamicValidation
+                  provideErrorHint
                 />
               </FieldErrorHint>
             </FieldBottomConstraints>
           </FieldProvider>
         </div>
         <div className={cx('confirm-new-password-field')}>
           <FieldProvider name="passwordRepeat">
-            <FieldErrorHint>
+            <FieldErrorHint provideHint={false}>
               <InputOutside
                 type={'password'}
                 icon={PasswordIcon}
                 maxLength={'256'}
                 placeholder={formatMessage(placeholders.confirmNewPassword)}
+                hasDynamicValidation
+                provideErrorHint
```

Comment:
I don't see the password validation message.

![image](https://user-images.githubusercontent.com/31861805/175539555-c3873ba4-d97e-4245-a77b-557d50ef3230.png)

Reply:
As I see in task, there is no error mesage for password

Reply:
Just to clarify that this is how it should be, because we had a message here earlier.